src/commands/appSettings/filterDownloadAppSettings.ts

```diff
@@ -11,42 +11,75 @@ import { localize } from "../../localize";
 export async function filterDownloadAppSettings(context: IActionContext, sourceSettings: { [key: string]: string }, destinationSettings: { [key: string]: string }, destinationName: string): Promise<void> {
     let suppressPrompt: boolean = false;
     let overwriteSetting: boolean = false;
+    let suppressPromptOfSettingsToIgnore = false;
```

Comment:
This variable is hard to read and get harder where its being used below in a condition due to the double negative.. can we instead change it to something like `showPromptForSettingsToIgnore` ? and update the relevant logic below?

Reply:
Yes sure!